examples/vello_editor/src/text.rs

```diff
@@ -39,6 +41,27 @@ impl Editor {
         self.editor.text()
     }
 
+    pub fn init(&mut self) {
+        self.start_time = Some(Instant::now());
```

Comment:
I think that the blinking should be reset if new text is added. Note how in e.g. Firefox the cursor stays a solid line whilst you are inputting text. It seems like it just toggles the cursor on, so the next blink interval would turn it off again? It's slightly unclear what the correct behaviour is, but just blinking all the time probably isn't it.

Reply:
I agree with that the blinking should be reset if the user are typing.
Thanks for the feedback!